turbopack/crates/turbopack-trace-server/src/span_ref.rs

```diff
@@ -268,6 +268,9 @@ impl<'a> SpanRef<'a> {
                     self_time += duration * duration / concurrent_time;
                 }
             }
+            if self.children().next().is_none() {
+                self_time = max(self_time, 1);
+            }
```

Comment:
This ensures that all spans are visible in the tracing

It might overweight spans smaller than 1µs, but that's an acceptable trade-off. (Maybe we should increase the resolution in the trace viewer to 0.1µs?)